### ISSUE TITLE
Detect dangling links in properties (previously known as frontmatter or YAML)

### ISSUE THREAD

**Issue by claremacrae** [graydon/obsidian-dangling-links#22]
Obsidian has introduced a [Properties](https://help.obsidian.md/Editing+and+formatting/Properties) UI that makes it easy to add links in the frontmatter/yaml.

It would be really helpful if this plugin could detect any dangling links in frontmatter.

Example file:

```
---
creation date: 2024-02-11 07:56
tags: 
aliases: 
delete me: "[[I Do not exist]]"
---

# Demo
```

No dangling link is shown:

![image](https://github.com/graydon/obsidian-dangling-links/assets/4840096/8eb81bdd-0fb2-411f-b285-73b1e75a0b81)

**Comment by claremacrae** [comment]
I've requested this facility in the Broken Links plugin:

https://github.com/ipshing/obsidian-broken-links/issues/5

**Comment by claremacrae** [comment]
Just to note that this is now available in [Broken Links release 1.2.0](https://github.com/ipshing/obsidian-broken-links/releases/tag/1.2.0) so closing.
